core/src/main/java/discord4j/core/object/util/PermissionSet.java

```diff
@@ -87,54 +98,126 @@ private PermissionSet(final long rawValue) {
     }
 
     /**
-     * Performs a logical AND of of this permission set with the other permission set.
+     * Performs a logical <b>AND</b> of this permission set with the other permission set.
+     * <p>
+     * The resultant set is the <b>intersection</b> of this set and the other set. A permission is contained iff it was
+     * contained in both this set and the other set.
+     * <pre>
+     * {@code
+     * PermissionSet set0 = PermissionSet.of(KICK_MEMBERS, BAN_MEMBERS);
+     * PermissionSet set1 = PermissionSet.of(KICK_MEMBERS);
+     * set0.and(set1) = PermissionSet.of(KICK_MEMBERS);
```

Comment:
This is an odd way to represent equality. Perhaps it should use `=>` or simply "is equivalent to"

Reply:
In my opinion, the equality sign isn't an odd way to represent equality.

Reply:
When it has a different meaning in the language (and the 2 directly preceeding lines) it is.